libV2/constants.js

```diff
@@ -26,11 +26,14 @@ module.exports = {
         'getBucketAcl',
         'getBucketCors',
         'getBucketLocation',
+        'getBucketObjectLock',
```

Comment:
I think these are added in one of the open PRs. Depending which merges first we may need to watch for duplicates.